core/tests/protractor_utils/LearnerDashboardPage.js

```diff
@@ -147,36 +219,36 @@ var LearnerDashboardPage = function() {
     expect(await feedbackMessage.first().getText()).toMatch(message);
   };
 
-  this.checkIncompleteExplorationSection = async function(explorationTitle) {
-    await this.navigateToInCompleteSection();
-    await this.navigateToIncompleteExplorationsSection();
-    await this.expectTitleOfExplorationSummaryTileToMatch(
-      explorationTitle);
-  };
-
-  this.checkCompleteExplorationSection = async function(explorationTitle) {
-    var completedSectionVisible = await (
-      completedExplorationsSection.isDisplayed());
-    if (!completedSectionVisible) {
-      await this.navigateToCompletedSection();
-    }
-    await this.navigateToCompletedExplorationsSection();
-    await this.expectTitleOfExplorationSummaryTileToMatch(
-      explorationTitle);
-  };
-
-  this.checkIncompleteCollectionSection = async function(collectionTitle) {
-    await this.navigateToInCompleteSection();
-    await this.navigateToIncompleteCollectionsSection();
-    await this.expectTitleOfCollectionSummaryTileToMatch(
-      collectionTitle);
-  };
-
-  this.checkCompleteCollectionSection = async function(collectionTitle) {
-    await this.navigateToCompletedSection();
-    await this.navigateToCompletedCollectionsSection();
-    await this.expectTitleOfCollectionSummaryTileToMatch(
-      collectionTitle);
+  this.navigateToCommunityLessonsAndCheckIncompleteExplorations = (
+    async function(explorationTitle) {
+      await this.navigateToCommunityLessonsSection();
+      await (await driver.findElement(by.css(
+        '.protractor-test-incomplete-community-lessons-section')));
```

Comment:
This doesn't do anything because we aren't doing anything with the value returned by `driver.findElement`. I think instead, you want to use chained element selectors in `this.expectTitleOfExplorationSummaryTileToMatch`. For example:

```js
var explorationTitle = element(
  by.css('.protractor-test-incomplete-community-lessons-section'
)).element(
  by.cssContainingText('.protractor-test-exp-summary-tile-title', title));
```

(note that this will also mean that `expectTitleOfExplorationSummaryTileToMatch` only applies to the incomplete community lessons section, so you'll need to create another function for the complete section)

ditto for the other cases where you use `driver.findElement`.

Reply:
Done. PTAL!